client/blocks/comment-detail/index.jsx

```diff
@@ -80,18 +81,40 @@ export class CommentDetail extends Component {
 
 	edit = () => noop;
 
+	getCommentObject = () => ( {
+		commentId: this.props.commentId,
+		isLiked: this.props.commentIsLiked,
+		postId: this.props.postId,
+		status: this.props.commentStatus,
+	} );
```

Comment:
Ah got it, this feels pretty similar to a redux action object (it's just missing the type), so we might even call this `getCommentStatusAction` where we might modify the fields to have both the last status and next status.